test/functional/transport/transport.transactions.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var _ = require('lodash');
+var crypto = require('crypto');
+var Promise = require('bluebird');
+
+var node = require('../../node.js');
+var ws = require('../../common/wsCommunication.js');
+var getTransaction = require('../../common/complexTransactions.js').getTransaction;
+var getTransactionPromisify = Promise.promisify(getTransaction);
+var getUnconfirmedTransaction = require('../../common/complexTransactions.js').getUnconfirmedTransaction;
+var getUnconfirmedTransactionPromisify = Promise.promisify(getUnconfirmedTransaction);
+
+function postTransaction (transaction, done) {
+	ws.call('postTransactions', {
+		transaction: transaction
+	}, done, true);
+}
+
+describe('Basic Transactions via websockets', function () {
+
+	var transaction;
+	var goodTransactions = [];
+	var badTransactions = [];
+	var account = node.randomAccount();
+
+	beforeEach(function () {
+		transaction = node.randomTx();
+	});
+
+	describe('schema', function () {
+	});
```

Comment:
Please put comments about why you are leaving it here. Or remove it entirely and leave comments about this being the right place to put schema definitions for the test run.